package.json

```diff
@@ -56,6 +56,7 @@
     "@babel/preset-env": "^7.8.6",
     "@babel/preset-react": "^7.8.3",
     "@babel/register": "^7.8.6",
+    "async-wait-until": "^1.2.4",
```

Comment:
This library (https://www.npmjs.com/package/async-wait-until) hasn't received an update for 2 years and doesn't appear to be used that much. If it's needed and there's not an alternative -- so be it. In general I try to keep our dependencies to a minimum and use only well-maintained and trusted libraries to reduce future problems.

Reply:
It turns out async-wait-until wasn't necessary, though the solution I found still seems brittle. The reason the library isn't maintained may be that it made its way into react-testing-library (https://testing-library.com/docs/react-testing-library/example-intro), which has a waitFor() function that would be handy!